app/views/crime_applications/edit.html.erb

```diff
@@ -48,6 +48,15 @@
         <p class="govuk-body">
           <%= @crime_application.applicant_dob %>
         </p>
+
+        <% if @crime_application.date_stamp.present? %>
```

Comment:
@willmcb and we also need to implement new logic relating to not showing date stamp if set but case type not stampable .

Reply:
Ah of course! 🤦